vspace/src/main/webapp/WEB-INF/views/exhibition/module.html

```diff
@@ -1,105 +1,97 @@
 <html layout:decorate="~{layouts/main}">
-
    <head>
       <style type="text/css">
          body {
          font-family: 'Montserrat', sans-serif;
          }
       </style>
+      <script th:inline="javascript">
+         $( document ).ready(function() {
+            $("#blocks a").css('color','rgb(150, 45, 62)').css('font-weight','bold');
+            $(".nav-bar").css('position','fixed').css('width','100%').css('z-index','125');
+         });
+             function openLangNav() {
+                 $("#myLangSidenav").css("width","240px");
+             }
+             function closeLangNav() {
+                 $("#myLangSidenav").css("width","0");
+             }
+             function openNav(){
+                 $("#mySidenav").css("width", "170px");
+                 $("#mySidenav").css("height", "700px");
+             }
+             function closeNav(){
+                 $("#mySidenav").css("width", "0px");
+             }
+        </script>
```

Comment:
what is this script for?

Reply:
I see that it was here bfore, but since there is not side navigation, is this needed?